include/fuzzuf/utils/check_parallel_mode_args.hpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * fuzzuf
+ * Copyright (C) 2022 Ricerca Security
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+/**
+ * @file check_parallel_mode_args.hpp
+ * @author Ricerca Security <fuzzuf-dev@ricsec.co.jp>
+ */
+#ifndef FUZZUF_INCLUDE_UTILS_CHECK_PARALLEL_MODE_ARGS_HPP
+#define FUZZUF_INCLUDE_UTILS_CHECK_PARALLEL_MODE_ARGS_HPP
+
+#include <boost/program_options.hpp>
+#include <fuzzuf/exceptions.hpp>
+#include <fuzzuf/utils/filesystem.hpp>
+#include <fuzzuf/utils/parallel_mode.hpp>
+#include <iostream>
+
```

Comment:
It's acceptable to place them under utils, but need to move them in the future.
```suggestion

// FIXME: this should be moved to algorithms/afl/?
```